index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo"> My Logo </a>
+        <div class="menu">
+          <img src="images/svg/Menu.svg" alt="Menu" />
+        </div>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Portfolio </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> About </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Contact </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="header">
+      <div class="container">
+        <h1 class="title">
+          I'm Eduardo <br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <a href="#" class="connect-button hover-blue-text">Let's connect</a>
```

Comment:
- I kindly suggest you use a semantic `hx` tag such as `h1, h2, h3` etc to display the highlighted element.

Reply:
Hello @Efiamotu-1 
First of all thanks for all your advice I really appreciate too much. 

I just have a question, I don't understand the last suggestion, do you try to tell me that instead to use the `p` tag,  I must use one of the `Hx` tags?

Thank you for your time.

Reply:
@DHE19 `hx` tags refers to the other heading tags in html such as `h1`, `h2`, `h3`, `h4`, `h5`,`h6`

Reply:
@Efiamotu-1  of course, but, I need to change the `p` tag, right?

Reply:
@DHE19 you are to wrap the word `let's connect` in the `hx` tag like the example below thats all
```html
<h2>LET'S CONNECT</h2>
```